status/flowcells.py

```diff
@@ -289,4 +289,16 @@ def post(self, flowcell):
                 self.set_header("Content-type", "application/json")
                 self.finish(json.dumps(links))
 
+#Functions
+def get_container_from_id(flowcell):
+    if flowcell[7:].startswith('00000000'):
```

Comment:
yes.... I saw it now.. I was a bit confused.... ok perfect.
Just wait for @senthil10 to see if there is another way to check the difference between MiSeq and HiSeq, I am not entirely sure that yours is "stable" for what "stable" means in this field....